fonts/stylesheet.css

```diff
@@ -0,0 +1,7 @@
+@font-face {
+    font-family: 'pt_serifbold';
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/FirstWebsait/pullRequest?prid=6170621)